src/docs/development/tools/devtools/code-size.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Using the code size tool
+description: Learn how to use the DevTools code size tool.
+---
+
+## What is it?
+
+The code size tool allows you to analyze the total size of your app, including
+both native and Dart code, or just the size of the Dart code in your app. You
+can view a single snapshot of "size information" using the [Analysis tab][], or
+compare two different snapshots of "size information" using the [Diff tab][].
+
+### What is "size information"?
+
+The Dart AOT compiler performs tree-shaking on your code when compiling your
+application. This means that the compiler attempts to optimize your app's size
+by removing pieces of code that are unused or unreachable.
+
+After the compiler optimizes your code as much as it can, the end result can be
+summarized as the collection of packages, libraries, classes, and functions that
+exist in the binary output, along with their size in bytes. This is the "size
+information" we can analyze in the code size tool to further optimize code and
```

Comment:
This is just part of what we analyze. We show the rest of the application package as well such as native code from a different compiler etc. If you meant to describe "Dart AOT size information" only, clarify in this and previous header.

Reply:
modified - please take a look